e2e/utils/get-env-info.ts

```diff
@@ -73,6 +73,22 @@ export function getNpmMajorVersion(): string {
   return npmMajorVersion;
 }
 
+export function getYarnMajorVersion(path: string): string {
+  try {
+    // this fails if path is not yet created
+    const [npmMajorVersion] = execSync(`yarn -v`, {
+      cwd: path,
+      encoding: 'utf-8',
+    }).split('.');
+    return npmMajorVersion;
+  } catch {
+    const [npmMajorVersion] = execSync(`yarn -v`, { encoding: 'utf-8' }).split(
+      '.'
+    );
+    return npmMajorVersion;
+  }
```

Comment:
this would be `yarnMajorVersion` yeah?
```suggestion
    const [yarnMajorVersion] = execSync(`yarn -v`, {
      cwd: path,
      encoding: 'utf-8',
    }).split('.');
    return npmMajorVersion;
  } catch {
    const [yarnMajorVersion] = execSync(`yarn -v`, { encoding: 'utf-8' }).split(
      '.'
    );
    return yarnMajorVersion;
  }
```